src/tests/eet/eet_test_cache.c

```diff
@@ -60,7 +60,13 @@ EFL_START_TEST(eet_test_cache_concurrency)
    Eina_Bool r;
    int tmpfd;
 
-   file = strdup("/tmp/eet_suite_testXXXXXX");
+   const char * filename = "eet_suite_testXXXXXX";
+   const char * tmpdir = eina_environment_tmp_get();
```

Comment:
Most of efl seems to prefer `type *var` over `type* var` or `type * var`. The last one looks like a multiplication operator =x